src/main/java/rip/bolt/ingame/utils/Messages.java

```diff
@@ -25,4 +29,18 @@ public static Component forfeit() {
                     command(
                         Style.style(NamedTextColor.YELLOW, TextDecoration.UNDERLINED), "forfeit")));
   }
+
+  public static Component matchLink(BoltMatch match) {
+    String url = AppData.Web.getMatch().replace("{matchId}", match.getId());
+
+    return text("Match link: ", NamedTextColor.WHITE)
+        .append(link(Style.style(NamedTextColor.BLUE, TextDecoration.UNDERLINED), url));
+  }
+
+  public static Component profileLink(MatchPlayer player) {
```

Comment:
Could have this shown on a command like /profile [player] but it feels a bit redundant

Reply:
Yeah, that was the plan wanted to run command ideas through you all first.

Not currently used but thought adding both at the same time to the code and config would make things easier.